pkg/vulnsrc/rocky/rocky.go

```diff
@@ -222,6 +222,7 @@ func (vs *VulnSrc) commit(tx *bolt.Tx, platformName string, errata []RLSA) error
 			input.PlatformName = platformName
 			input.CveID = cveID
 			input.Vuln = vuln
+			input.Erratum = erratum
```

Comment:
Can you please add a comment?

```suggestion
			input.Erratum = erratum // For Trivy Premium
```